model/build.gradle.kts

```diff
@@ -47,16 +47,11 @@ dependencies {
 }
 
 smallryeOpenApi {
-  infoTitle.set("Nessie API")
   infoVersion.set(project.version.toString())
-  infoContactName.set("Project Nessie")
-  infoContactUrl.set("https://projectnessie.org")
-  infoLicenseName.set("Apache 2.0")
-  infoLicenseUrl.set("http://www.apache.org/licenses/LICENSE-2.0.html")
   schemaFilename.set("META-INF/openapi/openapi")
   operationIdStrategy.set("METHOD")
   scanPackages.set(
-    listOf("org.projectnessie.api", "org.projectnessie.api.http", "org.projectnessie.model")
+    listOf("^(org.projectnessie.api(\\..+)?|org.projectnessie.api.http|org.projectnessie.model)$")
```

Comment:
Should the v1/v2 interfaces/classes reside in `org.projectnessie.api.v1`/`...api.v2` instead of `...apiv1`?

Reply:
I wanted to keep the "current" api as simply `api` with the idea of dropping `apiv1` once clients migrate to v2.

Do you see us supporting both versions for a long enough time to benefit from parallel `v1` and `v2` packages?

Reply:
On the other hand, it might be worth splitting java packages by API version anyway, just to make it clear at compile time... will give it a try :thinking:

Reply:
fixed